content/en/blog/_posts/XXXX-XX-XX-fine-grained-supplementalgroups-control-beta/index.md

```diff
@@ -0,0 +1,204 @@
+---
+layout: blog
+title: 'Kubernetes 1.33: Fine-grained SupplementalGroups control graduates to beta'
```

Comment:
We should update this based on https://kubernetes.io/docs/contribute/style/style-guide/#headings (and also version number should be prefixed with "v"):

```suggestion
title: 'Kubernetes v1.33: Fine-grained SupplementalGroups Control Graduates to Beta'
```